backend/substrapp/tasks/tasks.py

```diff
@@ -797,9 +802,38 @@ def _do_task(client, subtuple_directory, tuple_type, subtuple, compute_plan_id,
     with open(path.join(pred_path, 'perf.json'), 'r') as perf_file:
         perf = json.load(perf_file)
     result['global_perf'] = perf['all']
+
+    tag = subtuple.get("tag")
+    if tag and TAG_VALUE_FOR_TRANSFERT_BUCKET in tag:
+        try:
+            transfer_to_bucket(subtuple["key"], pred_path, model_path)
+        except ClientError as e:
+            logging.error(e)
+
     return result
 
 
+def transfer_to_bucket(tuple_key, pred_path, model_path):
```

Comment:
This file is almost 1k LoC. We should maybe extract this function in a kind of `exporter` class. I would also have a list of file as argument so it can be used to export predictions without the associated model for example. The signature might be `def export(name: str, paths: List[]) -> None:` and the class a S3Exporter in an exporters/ package or something similar.

Reply:
This is indeed a long file and the backend could use a bit of abstraction regarding inputs and outputs. But I believe this exceed this PR scope.
I will take the paths' list advise though. Thank you for the feedback.